pkg/docker/util.js

```diff
@@ -73,6 +73,20 @@ define([
             return cockpit.format(_("Exited $ExitCode"), state);
     };
 
+    util.render_container_restart_policy = function render_restart_policy(policy) {
+      switch (policy.Name) {
+        case "":
+          return "None";
+        case "on-failure":
+          var singular = policy.MaximumRetryCount == 1;
+          return "On Failure, retry " + policy.MaximumRetryCount + (singular ? ' time' : ' times');
+        default: /* http://stackoverflow.com/a/4878800 */
+          return policy.Name.replace('-', ' ').replace(/\w\S*/g, function(txt) {
+            return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
```

Comment:
This isn't translatable. Although this is nice and clean, it really needs to be broken out:

```
case "always":
    return _("Always");
```

And so on.